src/components/StateBar/__stories__/Playground.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import { ComponentStory } from '@storybook/react'
+import StateBar from '..'
+
+export const Playground: ComponentStory<typeof StateBar> = () => (
+  <StateBar>
+    <StateBar.State label="Label">your value</StateBar.State>
+    <StateBar.Bar value={50} />
+  </StateBar>
+)
```

Comment:
```suggestion
export const Playground: ComponentStory<typeof StateBar> = ({...props}) => (
  <StateBar {...props}>
    <StateBar.State label="Label">your value</StateBar.State>
    <StateBar.Bar value={50} />
  </StateBar>
)
```

Could we add props so we don't have `() => (...)` in the code snippet? 😄

Reply:
Done 👌